deployment/terraform/create.go

```diff
@@ -340,9 +341,167 @@ func (t *Terraform) PostProcessDatabase(extAgent *ssh.ExtAgent) error {
 		}
 	}
 
+	needsReboot, err := t.HasPendingRebootDBParams()
+	if err != nil {
+		return fmt.Errorf("failed to check whether the DB has pending-reboot parameters")
+	}
+
+	if needsReboot {
+		return t.RebootDBInstances(extAgent)
+	}
+
 	return nil
 }
 
+func (t *Terraform) HasPendingRebootDBParams() (bool, error) {
```

Comment:
On the other hand, are long files really that bad? Less jumping around, keep everything related in one place? 
0/5

Reply:
Never expected to see this comment :) I think "keep everything related in one place" is the keyword. We already have a db_operations.go that deals with RDS stuff so I think it would be good to keep RDS related stuff there.